utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -237,6 +273,46 @@ inline void set_descriptor_layout( RpptDescPtr srcDescPtr, RpptDescPtr dstDescPt
     }
 }
 
+// sets roi xywh values and dstImg sizes
+inline void set_roi(vector<string>::const_iterator imagePathsStart, vector<string>::const_iterator imagePathsEnd, RpptROI *roiTensorPtrSrc, RpptROI *roiTensorPtrDst, RpptImagePatchPtr dstImgSizes)
```

Comment:
Instead of set_roi, give better names, get_image_info or something like that
set_roi is slightly confusing

@r-abishek please share your thoughs on this

Reply:
Thats fine, is it a get function or set function? 
If it writes roi values probably use set, but something more descriptive  set_src_and_dat_roi()